src/System.Net.WebSockets.Client/src/Resources/Strings.resx

```diff
@@ -121,4 +121,10 @@
   <data name="net_securityprotocolnotsupported" xml:space="preserve">
     <value>The requested security protocol is not supported.</value>
   </data>
+  <data name="net_WebSockets_UWPClientCertSupportRequiresWindows10InsiderPreviewBuild16215OrGreater" xml:space="preserve">
+    <value>Support for client certificates in UWP requires Windows 10 Insider Preview Build 16215 or greater.</value>
```

Comment:
>"unsupported in Windows 10 version 1703 and earlier" 

I like that although I would suggest making it clearer by adding more text.

"This feature is unsupported in Windows 10 version 1703 and earlier versions. Please upgrade Windows 10 to a later release."